### PR TITLE
feat(locales): Added english translations

### PR DESCRIPTION
**Pull request by oliverstasa**
translation for admin

### CODE REVIEW

File: public/locales/en/hook.json [hunk 1]
```diff
@@ -0,0 +1,11 @@
+{
+	"imageService": {
+		"upload": "Image has been uploaded",
+
```
Comment (pchalupa) [inline public/locales/en/hook.json]: Please remove blank line.

**Review by pchalupa — CHANGES_REQUESTED** [review]

File: public/locales/en/hook.json (hunk 1 above)
Comment (oliverstasa) [reply public/locales/en/hook.json]: "upload"?

Comment (oliverstasa) [reply public/locales/en/hook.json]: oh you meen the blank line!

File: public/locales/en/page.json [hunk 2]
```diff
@@ -0,0 +1,24 @@
...
+		"logIn": "Login",
+		"username": "Username",
+		"password": "Password"
+	},
+	"gallery": {
+		"properties": "Properties",
+		"save": "Save",
+		"add": "Add",
+		"upload": "Upload",
+		"content": "Contect",
```
Comment (pchalupa) [inline public/locales/en/page.json]: Fix typo
```suggestion
		"content": "Content",
```

File: public/locales/en/page.json [hunk 3]
```diff
@@ -0,0 +1,24 @@
...
+		"title": "Title",
+		"description": "Description",
+		"detail": "Detail"
+	},
+	"image": {
+		"properties": "Properties"
+	},
+	"dashboard": {
+		"welcomeBack": "Wellcome back!"
```
Comment (pchalupa) [inline public/locales/en/page.json]: Fix typo
```suggestion
		"welcomeBack": "Welcome back!"
```

**Review by pchalupa — CHANGES_REQUESTED** [review]
Please fix typos

File: public/locales/en/page.json [hunk 4]
```diff
@@ -0,0 +1,24 @@
+{
+	"login": {
+		"login": "Login",
+		"logIn": "Login",
```
Comment (pchalupa) [inline public/locales/en/page.json]: Watch out! This was changed https://github.com/Stacha-dev/tardis-cms/blob/1d52abce17bf337cb04cc40b5e9177b45bd4305e/public/locales/cs/page.json#L6
```suggestion
		"submit": "Login",
```

**Review by pchalupa — APPROVED** [review]

**Pull request merged**